beetsplug/subsonic.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please include our standard license block comment at the top of the file. Then the documentation you've written below can go into a docstring instead of comments. (See some other plugins for examples.)